.github/scripts/transform_dbt_test_results.py

```diff
@@ -86,9 +86,6 @@
 import simplejson as json
 import yaml
 
-# Tests without a config.meta.category property will be grouped in
-# this default category
-DEFAULT_TEST_CATEGORY = "miscellaneous"
```

Comment:
I haven't deleted this variable, just moved it below to be closer to the other category-definition constant.